python/tvm/_ffi/libinfo.py

```diff
@@ -205,11 +205,18 @@ def find_include_path(name=None, search_path=None, optional=False):
         tvm_include_path = [os.path.join(p, "include") for p in header_path]
         dlpack_include_path = [os.path.join(p, "dlpack/include") for p in header_path]
         dmlc_include_path = [os.path.join(p, "dmlc-core/include") for p in header_path]
-
+        if use_nvcc:
+            cutlass_include_path = [os.path.join(p, "cutlass/include") for p in header_path]
```

Comment:
is it possible to make it configurable and move this to some cutlass specific places?